.github/actions/misspell/action.yml

```diff
@@ -21,9 +25,9 @@ runs:
       shell: bash
       run: |
           if [ -n "${{ inputs.EXCEPTIONS }}" ]; then
-              ./bin/misspell --error -i ${{ inputs.EXCEPTIONS }} . >misspell.log
+              ./bin/misspell --error -i ${{ inputs.EXCEPTIONS }} ${{ inputs.PATH }} >misspell.log
```

Comment:
is `inputs.PATH` work for you?
Shouldn't it be `inputs.path`?

Reply:
Test run here: https://github.com/PelionIoT/mbed-edge-examples/actions/runs/12349184886
I'm not sure the arguments are case sensitive.